src/test/java/teammates/test/cases/webapi/GetSessionResultsActionTest.java

```diff
@@ -0,0 +1,130 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.EntityNotFoundException;
+import teammates.common.exception.UnauthorizedAccessException;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.GetSessionResultsAction;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.output.SessionResultsData;
+import teammates.ui.webapi.request.Intent;
+
+/**
+ * SUT: {@link GetSessionResultsAction}.
+ */
+public class GetSessionResultsActionTest extends BaseActionTest<GetSessionResultsAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.RESULT;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() throws Exception {
```

Comment:
Where are the tests for student's accessing their results?

Reply:
I have added them in the latest commit. Thanks for spotting that!